src/components/Search.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) => {
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
         placeholder={"Search your Recent Transactions"}
-      />
+
+
+        onChange={ (event) => props.handleChange(event)}/>
```

Comment:
don't forget to bind the value of this input. If you have an `onChange` you also want to bind the value to whatever it's changed to.